packages/runtime/src/useCases/consumption/requests/AcceptIncomingRequest.ts

```diff
@@ -14,7 +14,7 @@ export class AcceptIncomingRequestUseCase extends UseCase<AcceptIncomingRequestR
     }
 
     protected async executeInternal(request: AcceptIncomingRequestRequest): Promise<Result<LocalRequestDTO, ApplicationError>> {
-        let localRequest = await this.incomingRequestsController.getIncomingRequest(CoreId.from(request.requestId));
+        let localRequest = await this.incomingRequestsController.getIncomingRequestWithUpdatedExpiry(CoreId.from(request.requestId));
```

Comment:
It looks like it doesn't make sense to call `getIncomingRequest` ever again, why don't you add the functionality from `getIncomingRequestWithUpdatedExpiry` to `getIncomingRequest`

same for outgoing and so on :)

Reply:
This was a renaming of the `getIncomingRequest` which always had the side-effect of expiration updating. I made that explicit because it's used for the expiration update if creating a relationship with an expired template fails, which looks very weird if there's only a `get`.
This reason doesn't exist for outgoing requests but I guess I did it to be uniform.

Reply:
urgh